test/registry/systemd/registry.container

```diff
@@ -4,7 +4,7 @@ Description=Registry container
 [Container]
 Label=app=registry
 ContainerName=registry
-Image=docker.io/library/registry:latest
+Image=docker.io/library/registry:2
```

Comment:
So dockerhub is usually a pain in CI, unless you have a subscription. We went through all of it a few years ago and moved stuff to quay. https://github.com/kubevirt/containerized-data-importer/pull/2664

Reply:
Right, this issue already exits in the current code. I can add a commit using libpod image.